vertical_integration/src/aspen_to_heron.py

```diff
@@ -0,0 +1,66 @@
+"""
```

Comment:
I think the conversion scripts should all be moved to a `vertical_integration/scripts` folder and the function that do the conversion should be moved into the respective script from `main.py`

Also, all scripts that will be run by a user in the command line should start with a `#!/usr/bin/env python` line

Reply:
Done